frontend/mock-backend/fixed-data.ts

```diff
@@ -22,6 +22,7 @@ import jsonRun from './json-runtime';
 import coinflipRun from './mock-coinflip-runtime';
 import errorRun from './mock-error-runtime';
 import xgboostRun from './mock-xgboost-runtime';
+import suspenseRun from './suspense-runtime';
```

Comment:
Should this be "suspend" instead of suspense?

**Suspense** is a state of mental uncertainty, anxiety, of being undecided, or of being doubtful. In a dramatic work, suspense is the anticipation of the outcome of a plot or of the solution to an uncertainty, puzzle, or mystery, particularly as it affects a character for whom one has sympathy.

**Suspend** means temporarily prevent from continuing or being in force or effect.